ui/keycode.js

```diff
@@ -0,0 +1,46 @@
+/*!
+ * jQuery UI Keycode @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ */
+
+//>>label: Keycode
+//>>group: Core
+//>>description: Provide keycodes as keynames
+//>>docs: http://api.jqueryui.com/category/keycodes/
```

Comment:
Still wrong URL, the current documentation is at http://api.jqueryui.com/jQuery.ui.keyCode/